src/components/posts/PostLike.js

```diff
@@ -0,0 +1,67 @@
+import like from './like.png';
+import React from 'react';
+import { useState, useEffect } from 'react';
+import client from '../../utils/client';
+import './style.css';
+import colours from '../../utils/colours';
+
+const PostLike = ({likes=[], postId}) => { 
+
+const blue = colours.blue;
+const grey = colours.grey;
+
+const [divStyle, updateStyle] = useState({ backgroundColor: grey });
```

Comment:
Is there a reason these names don't follow the usual convention of "state", "setState" ? I.e. "divStyle", "setDivStyle"? or "postLikes", "setPostLikes"?

Reply:
There isn't, will stick to this going forward.